src/openms/include/OpenMS/KERNEL/RangeManager.h

```diff
@@ -126,12 +126,20 @@ struct OPENMS_DLLAPI RangeBase
   /// only useful if isEmpty() returns false
   double getMin() const
   {
+    if (isEmpty())
+    {
```

Comment:
document that all these methods (also the alternative names, e.g. getMinMZ) throw now.